### PR TITLE
HARMONY-895: Implement deflate shuffle compression

### PR DESCRIPTION
**Pull request by bilts**
The notebook in the PR (for Summer ESIP) shows this working

### CODE REVIEW

**Comment by bilts** [comment]
Fixes #5

File: eosdis_store/dmrpp.py [hunk 1]
```diff
@@ -210,8 +214,15 @@ def array_to_zarr(node, dims, prefix=''):
     # NOTE - this is null in test file
     zarray['fill_value'] = zattrs.get('_FillValue')
 
-    #if node.attrib['name'] == 'time':
-    #    set_trace()
+    # Automatic scale factor and offset filter.  Not yet reliable
```
Comment (indiejames) [inline eosdis_store/dmrpp.py:217]: Not yet reliable?

Comment (indiejames) [reply eosdis_store/dmrpp.py:217]: Do you have a ticket to make this work? If so you should add it to the comment.

**Review by indiejames — APPROVED** [review]

**Review by vinnyinverso — APPROVED** [review]

File: eosdis_store/dmrpp.py (hunk 1 above)
Comment (bilts) [reply eosdis_store/dmrpp.py:217]: It only works in some special cases when variable data types don't change. I'll clarify and add a ticket and ticket number

**Pull request merged**